src/app/pipes/shortumber.pipe.ts

```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'shortNumber',
+})
+export class ShortNumberPipe implements PipeTransform {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  transform(string: string | undefined, args?: any): string | null {
```

Comment:
I think we could simplify implementation transform method 
Possible implementation example:
const formatCounts = (counts) => {
const sumDigits = counts.split('').length;
if(sumDigits >= 10) return counts.slice(0, -9) + 'B';
if(sumDigits >= 7) return counts.slice(0, -6) + 'M';
if(sumDigits >= 4) return counts.slice(0, -3) + 'K';
return counts;
} 
It seems to me This implementation more readble.

Reply:
@AnatoliRub  please use 
<img width="209" alt="image" src="https://user-images.githubusercontent.com/20155344/154475139-3989553b-a6be-4a38-8180-e1b93a396fcc.png">

Reply:
```suggestion
  const formatCounts = (counts) => {
const sumDigits = counts.split('').length;
if(sumDigits >= 10) return counts.slice(0, -9) + 'B';
if(sumDigits >= 7) return counts.slice(0, -6) + 'M';
if(sumDigits >= 4) return counts.slice(0, -3) + 'K';
return counts;
}
```